src/lorentzvectors.cxx

```diff
@@ -115,5 +115,75 @@ ROOT::RDF::RNode buildMet(ROOT::RDF::RNode df, const std::string &met_pt,
     };
     return df.Define(outputname, construct_metvector, {met_pt, met_phi});
 }
+/**
+ * @brief Function used to construct the missing transverse energy lorentz
+ * vector from the dileptonsystem from the Higgs boson in the full hadronic channel.
+ *
+ * @param df the input dataframe
+ * @param outputname name of the new column containing the missing transverse
+ * energy lorentz vector.
+ * @param inputvectors a vector of the two names of the columns containing the required lorentz vectors
+ * @return a new df, containing the new column
+ */
+/// Function to calculate the neutrino four vector from a pair of lorentz vectors (visible Higgs system) and
+/// add it to the dataframe. see AN(HIG-19-010)
+///
+/// \param df the dataframe to add the quantity to
+/// \param outputname name of the new column containing the pt value
+
+///
+/// \returns a dataframe with the new column
+ROOT::RDF::RNode p4_miss_tt(ROOT::RDF::RNode df, const std::string &outputname,
+                        const std::vector<std::string> &inputvectors) {
+    return df.Define(
+        outputname,
+        [](const ROOT::Math::PtEtaPhiMVector &p4_1,
+           const ROOT::Math::PtEtaPhiMVector &p4_2) {
+            if (p4_1.pt() < 0.0 || p4_2.pt() < 0.0)
+                return ROOT::Math::PtEtaPhiEVector(default_float,default_float,default_float,
+                default_float);;
+            auto dileptonsystem = p4_1 + p4_2;
+            auto dileptonsystem_scaled = 0.47*dileptonsystem;
```

Comment:
why not use a more generic implementation, where you can configure this correction factor ? From what I see, the lt and tt functions are identical apart from the 0.47 and the 0.69